data/json/items/fluff.json

```diff
@@ -235,5 +235,167 @@
+    "material": [ "steel" ],
+    "flags": [ "TRADER_AVOID", "UNBREAKABLE_MELEE", "CONDUCTIVE" ],
+    "volume": "846 ml",
+    "longest_side": "218 mm",
+    "bashing": 4,
+    "ammo_type": "thrown",
+    "thrown_damage": [ { "damage_type": "bash", "amount": 2 } ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "dumbbell_10lb",
+    "name": { "str": "10lb dumbbell" },
+    "description": "A decent sized dumbbell. It is too heavy to throw any significant distance. Looks like a good bashing object.",
+    "weight": "4536 g",
+    "price_postapoc": 50,
+    "price": "999 cent",
+    "color": "black",
+    "symbol": "|",
+    "material": [ "steel" ],
+    "flags": [ "TRADER_AVOID", "UNBREAKABLE_MELEE", "ALWAYS_TWOHAND", "CONDUCTIVE" ],
+    "volume": "1719 ml",
+    "longest_side": "245 mm",
+    "bashing": 7
+  },
+  {
+    "type": "GENERIC",
+    "id": "dumbbell_20lb",
+    "name": { "str": "20lb dumbbell" },
+    "description": "A heavy piece of metal with hexagonal block ends used to workout. It is awkward to use as a weapon, but it can still do the job.",
+    "weight": "9072 g",
+    "to_hit": -1,
+    "price_postapoc": 100,
+    "price": "1995 cent",
+    "color": "black",
+    "symbol": "|",
+    "material": [ "steel" ],
+    "techniques": [ "BRUTAL" ],
+    "flags": [ "SLOW_WIELD", "TRADER_AVOID", "UNBREAKABLE_MELEE", "ALWAYS_TWOHAND", "MESSY", "CONDUCTIVE" ],
+    "volume": "3890 ml",
+    "longest_side": "304 mm",
+    "bashing": 10
+  },
+  {
+    "type": "GENERIC",
+    "id": "plate_2_5lb",
+    "name": { "str": "2.5lb plate" },
+    "description": "A very light disc shaped weight used on a weight lifting bar. Can be used as a bashing weapon or you could throw it.",
+    "weight": "1134 g",
+    "to_hit": 2,
+    "price_postapoc": 12,
+    "price": "249 cent",
+    "color": "black",
+    "symbol": "o",
+    "material": [ "steel" ],
+    "flags": [ "TRADER_AVOID", "UNBREAKABLE_MELEE", "CONDUCTIVE" ],
+    "volume": "255 ml",
+    "longest_side": "160 mm",
+    "bashing": 3,
+    "ammo_type": "thrown",
+    "thrown_damage": [ { "damage_type": "bash", "amount": 7 } ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "plate_5lb",
+    "name": { "str": "5lb plate" },
+    "description": "A light disc shaped weight used on a weight lifting bar. Can be used as a bashing weapon or you could throw it.",
+    "weight": "1134 g",
+    "to_hit": 1,
+    "price_postapoc": 23,
+    "price": "449 cent",
+    "color": "black",
+    "symbol": "o",
+    "material": [ "steel" ],
+    "flags": [ "TRADER_AVOID", "UNBREAKABLE_MELEE", "CONDUCTIVE" ],
+    "volume": "465 ml",
+    "longest_side": "99 mm",
+    "bashing": 4,
+    "ammo_type": "thrown",
+    "thrown_damage": [ { "damage_type": "bash", "amount": 11 } ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "plate_10lb",
+    "name": { "str": "10lb plate" },
+    "description": "A decently weighted disc shaped weight used on a weight lifting bar. Can be used as a bashing weapon. It is to awkward to throw.",
```

Comment:
```suggestion
    "description": "A decently weighted disc shaped weight used on a weight lifting bar. Can be used as a bashing weapon. It is too awkward to throw.",
```

Reply:
Use double sentence spacing after periods. [doc/MANUAL_OF_STYLE.md](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md)

Reply:
10-4 buddy. I'm on it this weekend. Thanks for your suggestions.